### DISCUSSION TITLE
Docker scheduler image required?

### DISCUSSION THREAD

**Discussion by web2wire** [photoprism/photoprism#1722] [Q&A] [answered]
The docker compose file at https://dl.photoprism.org/docker/docker-compose.yml lists only the database and app containers whereas this one https://dl.photoprism.org/docker/scheduler/docker-compose.yml also lists a "required" ofelia container. Is this actually required or just a nice to have for automatic indexing etc?

Thanks

Phil

**Comment by lastzero** [comment, marked as answer]
Yes, ofelia is required to run jobs. If you don't want a job runner / scheduler you can use our regular example without Ofelia. Note you also need to add a container name for photoprism like `container_name: photoprism` so that Ofelia knows which container to use for running the jobs.
